src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml

```diff
@@ -69,35 +69,35 @@ limitations under the License.
 
     <command source="client" code="0x04" name="GetProgramGuide" response="ProgramGuideResponse" optional="true">
       <description>This command retrieves the program guide. It accepts several filter parameters to return specific schedule and program information from a content app. The command shall receive in response a ProgramGuideResponse.</description>
-      <arg name="StartTime" type="epoch_s" optional="true"/>
-      <arg name="EndTime" type="epoch_s" optional="true"/>
-      <arg name="ChannelList" type="ChannelInfoStruct" array="true" optional="true"/>
-      <arg name="PageToken" type="PageTokenStruct" optional="true"/>
-      <arg name="RecordingFlag" type="RecordingFlagBitmap" optional="true"/>
-      <arg name="ExternalIDList" type="AdditionalInfoStruct" array="true" optional="true"/>
-      <arg name="Data" type="octet_string" optional="true"/>
+      <arg id="0" name="StartTime" type="epoch_s"/>
+      <arg id="1" name="EndTime" type="epoch_s"/>
+      <arg id="2" name="ChannelList" type="ChannelInfoStruct" array="true" optional="true" length="255"/>
+      <arg id="3" name="PageToken" type="PageTokenStruct" optional="true" isNullable="true"/>
+      <arg id="5" name="RecordingFlag" type="RecordingFlagBitmap" optional="true" isNullable="true" min="0x00" max="0x07"/>
```

Comment:
is min, max, isNullable supported?

Reply:
IsNullable is definitely supported. I don't think the min/max does anything, but isn't hurting.